svthalia/concrexit
# Re-enable HTML in registration status message

Closes #3515.



### Summary
Fairly trivial, bleach the registration status message so that e.g. links here work again

### How to test

1. Create an event without registrations
2. Set the message so that it contains a link
3. Observe that the link appears in the browser
